lib/langchain/tool/news_retriever.rb

```diff
@@ -20,9 +20,9 @@ class NewsRetriever
       property :from, type: "string", description: "A date and optional time for the oldest article allowed. This should be in ISO 8601 format"
       property :to, type: "string", description: "A date and optional time for the newest article allowed. This should be in ISO 8601 format"
       property :language, type: "string", description: "The 2-letter ISO-639-1 code of the language you want to get headlines for", enum: ["ar", "de", "en", "es", "fr", "he", "it", "nl", "no", "pt", "ru", "sv", "ud", "zh"]
-      property :sort_by, type: "string", description: "The order to sort the articles in", enum: ["relevancy", "popularity", "publishedAt"]
       property :page_size, type: "integer", description: "The number of results to return per page (request). 5 is the default, 100 is the maximum"
       property :page, type: "integer", description: "Use this to page through the results if the total results found is greater than the page size"
+      property :sortBy, type: "string", description: "The order to sort the articles in", enum: ["relevancy", "popularity", "publishedAt"]
```

Comment:
I guess I would ask this question: If we're telling the LLM that the parameter is called "sort_by" then why is it sending a function call with "sortBy"? I've not seen it happen with gpt-4 but I have seen this happen with Gemini. 

It's fascinating that you can tell it about it's issue and it'll resend a call with the corrected param:
![image](https://github.com/user-attachments/assets/63de9a40-0ba8-43e6-acab-0d5c16d6ea87)

Reply:
@Rukomoynikov The answer might just be "Gemini is hallucinating" but I hope that's not the answer.

Reply:
Hi @andreibondarev 

I think I found the issue, but didn't solve.

The sort_by parameters gets covernted into sortBy here in the function below. I think Google is not hallucinating, the langchain lib sends instructs Gemini to use sortBy parameter. And I believe it may happen with other `NewsRetriever` params aka search_in, exclude_domains, page_size but it's not noticeable now as Google doesn't use it.
https://github.com/patterns-ai-core/langchainrb/blob/ff699356068bd7d6bc768e5518f6b4fdcbdfc90f/lib/langchain/llm/google_gemini.rb#L65

Reply:
This PR fixes the issue, though it's not the most elegant solution.

Reply:
Sorry for bothering you @andreibondarev maybe I could improve the solution? Do you think the direction is wrong?